### PR TITLE
Design Manual IA

### PR DESCRIPTION
**Pull request by jenn-franklin**
## Changes

• Changed main nav and reorganized pages
• Matched file names to side-nav labels
• Added redirects to point outdated URLs to new URLs
## Review

@Scotchester
For content, one of the following: @kurzn @sonnakim @ielerol @stephanieosan 
## Link for viewing

https://huetingj.github.io/design-manual/
## Screenshots

screen shot 2016-05-15 at 11 12 33 pm

screen shot 2016-05-15 at 11 12 42 pm

screen shot 2016-05-15 at 11 12 51 pm

screen shot 2016-05-15 at 11 13 00 pm

screen shot 2016-05-15 at 11 18 32 pm

### CODE REVIEW

**Comment by stephanieosan** [comment]
This looks good from a content perspective! 

I think we'll have to figure out whether we have a clash between the logo and our navigation once we add Data Visualization, but we can figure that out when the time comes. 

@Scotchester Do you have time to take a look at this PR from a code perspective in the next couple of days? If not, we can ask @cfarm or @virginiacc to take a look. 

Getting this merged in is a blocker for the V1 folks to start PR'ing their pages. cc: @duelj @ajbush

**Comment by Scotchester** [comment]
@stephanieosan Sorry, I will not. I'm swamped with finishing up the Auto Loans site.

File: best-practices/accessibility-best-practices.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ---
 layout: page
-title: Accessible interfaces
-category: Guides
+title: Accessibility best practices
+category: Best practices
+redirect_from: "http://cfpb.github.io/design-manual/guides/accessible-interfaces.html"
```
Comment (virginiacc) [inline best-practices/accessibility-best-practices.md]: I think we can just specify the path in `redirect_from` (ie `redirect_from: "/guides/accessible-interfaces.html"`). That makes the redirects work for me locally...

File: 2 [hunk 2]
```diff
@@ -0,0 +1,7 @@
+ Merge branch 'gh-pages' of github.com:cfpb/design-manual into gh-pages
+
+# Please enter a commit message to explain why this merge is necessary,
+# especially if it merges an updated upstream into a topic branch.
+#
+# Lines starting with '#' will be ignored, and an empty message aborts
+# the commit.
```
Comment (virginiacc) [inline 2]: Is this file a side effect of a merge?

**Comment by virginiacc** [comment]
@huetingj This all looks good to me! 👍  I made one suggestion about the structure of the redirects so they work locally. Other than that, I think it's ready to merge -- although it looks like I don't have access to do so...

**Comment by jenn-franklin** [comment]
thank you @virginiacc! i'll restructure the redirect this afternoon.

**Comment by virginiacc** [comment]
@huetingj The updates look great! I don't think file `2` is necessary -- it looks like it contains a commit message -- but if people are waiting on this PR we could merge now & take it out later...

**Comment by jenn-franklin** [comment]
Thanks @virginiacc! I deleted the file. @Scotchester this should be ready to merge. Thank you!

**Pull request merged**